tests/forms_tests/tests/test_forms.py

```diff
@@ -3014,6 +3014,134 @@ class UserRegistration(Form):
             "</td></tr>",
         )
 
+        def test_help_text_aria_describedby(self):
```

Comment:
I don't understand why we need this new test vs ensuring that the test above `test_help_text` includes the proper `aria-describedby` attribute. Am I missing something?

Reply:
You are correct. The separate test in indeed not needed and is removed.